Moli/Core/Screens/EditProfile/EditProfileView.swift

```diff
@@ -1,70 +1,211 @@
+            ]
+        }
     }
 
+
     var body: some View {
-        VStack {
-            Form {
-                Section {
-                    TextField(myUser.user.profile.streamerName, text: $editProfileVm.editedStreamerName)
-                    TextField("@\(myUser.user.profile.streamerHandle)", text: $editProfileVm.editedStreamerHandle)
-                    TextField(myUser.user.profile.biography ?? "Enter biography", text: $editProfileVm.editedBiography)
-                    TextField(
-                        myUser.user.profile.websiteUrl ?? "Enter a website url",
-                        text: $editProfileVm.editedWebsiteUrl)
+        ScrollView(showsIndicators: false) {
+            ProfileAndBannerImageView(
+                bannerImages: myUser.user.profile.bannerImages,
+                profileUser: profileUserImage(),
+                isMe: true,
+                isEditProfileMode: true
+            )
+            VStack(alignment: .leading) {
+                Text("About you")
+                    .font(.labelSmall)
+                    .foregroundColor(.gray500)
+                LabelTextField(
+                    maxLimit: editProfileViewModel.STREAMER_NAME_MAX,
+                    minLimit: editProfileViewModel.STREAMER_NAME_MIN,
+                    label: "Name",
+                    text: $editProfileViewModel.editedStreamerName)
+                    .padding(.bottom, 8)
+                LabelTextField(
+                    maxLimit: editProfileViewModel.STREAMER_HANDLE_MAX,
+                    minLimit: editProfileViewModel.STREAMER_HANDLE_MIN,
+                    label: "Username",
+                    text: $editProfileViewModel.editedStreamerHandle)
+                    .padding(.bottom, 8)
+                VStack(alignment: .trailing) {
+                    LabelTextEditor(
+                        maxLimit: editProfileViewModel.BIOGRAPHY_MAX,
+                        label: "Bio",
+                        text: $editProfileViewModel.editedBiography,
+                        keyboardType: .twitter)
+                        .frame(minHeight: 80, alignment: .leading)
                 }
-                Section {
-                    AsyncButton(action: updateProfile) {
-                        Text("Update profile")
-                    }
+                .padding(.bottom, 16)
+
+                Text("Link")
+                    .font(.labelSmall)
+                    .foregroundColor(.gray500)
+
+                LabelTextField(
+                    maxLimit: editProfileViewModel.WEBSITE_URL_MAX,
+                    placeHolder: "Link website",
+                    text: $editProfileViewModel.editedWebsiteUrl)
+                    .padding(.bottom, 8)
+
+                Text("Social")
+                    .font(.labelSmall)
+                    .foregroundColor(.gray500)
+
+                if hasInstagram {
+                    SocialFieldView(label: "Unlink instagram", imageName: "Instagram")
```

Comment:
:bomb: Jag hojtade Aida för en UX fix här, och det var om att visa själva display name:n man har på Instagram istället för "Unlink instagram"

```suggestion
                    SocialFieldView(label: "@\(instagramDisplayName)", imageName: "Instagram")
```